submodules/LIB/hardware/syn/genus/build.mk

```diff
@@ -0,0 +1,61 @@
+# (c) 2022-Present IObundle, Lda, all rights reserved
+#
+# This makefile is used at build-time
```

Comment:
This makefile segment

Reply:
leave or remove?

Reply:
It is not a makefile. The .mk are makefile **segments**

Reply:
changed to makefile segment